### ISSUE TITLE
MS EDGE formatting issue

### ISSUE THREAD

**Issue by luisrudge** [evgenyrodionov/redux-logger#170]
MS EDGE doesn't support the same syntax as chrome, so my logging messages are all messed up :(

![image](https://cloud.githubusercontent.com/assets/941075/17375326/d280e612-5986-11e6-8219-dd56100ab9ae.png)

**Comment by evgenyrodionov** [comment]
Yeah, I see it too, it's very weird. Is anyone know how to fix this?

**Comment by peteruithoven** [comment]
Redux-logger uses styles in console.log's to enable colors, see: 
https://developer.mozilla.org/en-US/docs/Web/API/console#Styling_console_output
http://stackoverflow.com/questions/7505623/colors-in-javascript-console

You can customize the colors, see: 
https://github.com/evgenyrodionov/redux-logger#api
By default they seem to be: 
``` js 
colors: {
  title: () => "#000000",
  prevState: () => "#9E9E9E",
  action: () => "#03A9F4",
  nextState: () => "#4CAF50",
  error: () => "#F20404",
}
```
You could disable al colors (styling) with:
``` js
colors: {}
```

**Comment by peteruithoven** [comment]
I also notices that Firefox doesn't support styles in `console.group` calls, while it does support them in `console.log`.

Someone did create an abstraction, but it doesn't do styling in Firefox.
https://github.com/icodeforlove/Console.js
http://stackoverflow.com/questions/16485093/any-way-of-detecting-whether-a-browsers-console-is-able-to-render-colors

I tried searching on npm, but packages mostly focus on usage in node.js, like chalk. Or [console.style](https://github.com/daniellmb/console.style) for example doesn't seem to support groups.

**Comment by peteruithoven** [comment]
We could add checks ourselves, maybe something like:
https://esnextb.in/?gist=685a3bb8c9c2d9f04db94c0c37524ce6

**Comment by jdalton** [comment]
`/\bEdge\b/.test(navigator.userAgent)`

**Comment by wallacyyy** [comment]
Is someone looking into this ? Is a nice feature, nowadays the color formatting does not look good on Firefox. The PR is quite old, someone will still review and merge it or a new one should be created ?

**Comment by pelotom** [comment]
This is also an issue when running in Jest.

**Comment by huyz** [comment]
Might as well close this issue.  No one is going to waste time fixing formatting for Edge when Microsoft is switching to Chromium.
